product_docs/docs/pgd/4/rel_notes/pgd_4.3.1_rel_notes.mdx

```diff
@@ -0,0 +1,28 @@
+navTitle: "Version 4.3.1"
+---
+
+EDB Postgres Distributed version 4.3.1 is a minor release of EDB Postgres Distributed 4, which includes bug fixes for issues identified in previous versions.
+
+!!! Note
+    This version is required for EDB Postgres Advanced Server versions 12.15, 13.11, 14.8 and later.
+
+| Component | Version |  Type           |                                                   Description                                                           |
+| --------- | ------- | --------------- | ------------------------------------------------------------------------------------------------------------------------|
+| BDR       | 4.3.1   | Feature         | Add pid to the log message emitted upon writer process death (BDR-2534) |
+| BDR       | 4.3.1   | Feature         | Make Raft leader step down on loss of quorum  (isolated) |
+| BDR       | 4.3.1   | Bug fix         | Fix memory leak in consensus process (BDR-3409, RT91830) |
+| BDR       | 4.3.1   | Bug fix         | Fix issue where a node can be inconsistent with the group after rejoining. If a node had been part of a sub-group, then parted and then rejoined to the group, it could be inconsistent with the group. The changes from some nodes of the group would be replayed from a wrong starting point, resulting in potential data-loss. |
+| BDR       | 4.3.1   | Bug fix         | Fix join and replication when SDW and standby_slot_names are set (BDR-3176, RT89702, RT89536) |
+| BDR       | 4.3.1   | Bug fix         | Fix spurious warnings when processing sequence options (BDR-3295, RT90668) |
+| BDR       | 4.3.1   | Bug fix         | Fix upgrades for nodes with CRDTs (BDR-3302) |
+| BDR       | 4.3.1   | Bug fix         | Fix replication for sub group nodes (BDE-3204, RT89814)  If a node had been part of a sub-group, then parted and then rejoined to the group, it could be inconsistent with the group. The changes from some nodes of the group would be replayed from a wrong starting point, resulting in potential data-loss. |
+| BDR       | 4.3.1   | Bug fix         | Adjust lag tracking parameters for LCRs from pg_stat_replication (BDR-3040) |
+| BDR       | 4.3.1   | Bug fix         | Convert all sequences to "distributed" during create node (BDR-2890) |
+| BDR       | 4.3.1   | Bug fix         | Fix streaming transactions with standby_slot_names (BDR-3248)  This could have led to a subscriber-only node get ahead of a data-node.|
+| BDR       | 4.3.1   | Bug fix         | Reload configuration for the pglogical receiver (BDR-2337)  Reload and apply the configuration changes when the server receives reload signal|
+| BDR       | 4.3.1   | Enhancement     | Avoid restarting sync workers.(BDR-3415)  This is to prevent the node join from failing when config changes are made that signal the restart of subscription workers. |
+| CLI       | 1.1.1   | Change          | Upgrade 3rd party dependencies to fix Github dependabot alerts |
```

Comment:
@Jagdish-Kewat-EDB I don't think this should get in. Is this of customer interest?

Reply:
Yes, it's a security alert and the only change for CLI 1.1.1